contrib/scraper_joao_pessoa.py

```
@@ -0,0 +1,112 @@
+            },
+            dont_filter=True,
+        )
+
+    async def parse(self, response):
+        page = response.meta["playwright_page"]
+
+        try:
+            await page.wait_for_selector(
+                "table, .ninja-table, [class*='ninja'], [id*='ninja']", timeout=20000
+            )
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)